Tests/Rules/UseConsistentWhitespace.tests.ps1

```diff
@@ -243,10 +255,132 @@ $x = @{
         It "Should not find a violation if a end of input follows a semi-colon" {
             $def = @'
 $x = "abc";
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
+        }
+    }
+
+
+    Context "CheckPipe" {
+        BeforeAll {
+            $ruleConfiguration.CheckInnerBrace = $false
+            $ruleConfiguration.CheckOpenBrace = $false
+            $ruleConfiguration.CheckOpenParen = $false
+            $ruleConfiguration.CheckOperator = $false
+            $ruleConfiguration.CheckPipe = $true
+            $ruleConfiguration.CheckSeparator = $false
+        }
+
+        It "Should find a violation if there is no space after pipe" {
+            $def = 'Get-Item |foo'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            Test-CorrectionExtentFromContent $def $violations 1 '' ' '
+        }
+
+        It "Should find a violation if there is no space before pipe" {
+            $def = 'Get-Item| foo'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            Test-CorrectionExtentFromContent $def $violations 1 '' ' '
+        }
+
+        It "Should not find a violation if there is 1 space before and after a pipe" {
+            $def = 'Get-Item | foo'
```

Comment:
should we be checking for things like `Get-Item      |     foo`?

Reply:
Yes, I'll add 2 test cases for that as well.